kombu/connection.py

```diff
@@ -658,7 +657,7 @@ def _info(self, resolve=True):
 
     def info(self):
         """Get connection info."""
-        return OrderedDict(self._info())
+        return dict(self._info())
```

Comment:
can you find out some more OrderedDict in the code base please?

Reply:
Will do. There are more.